utils.sh

```diff
@@ -67,7 +67,7 @@ function create_cluster() {
 
     find assets/generated -name '*.yaml' -exec cp -f {} ${assets_dir}/openshift \;
 
-    if [[ "${IP_STACK}" == "v4v6" && "$OPENSHIFT_VERSION" =~ 4.[67] ]]; then
+    if [[ "${IP_STACK}" == "v4v6" && "$(openshift_version $OCP_DIR)" =~ 4.[67] ]]; then
```

Comment:
Ah good catch, I wonder if we can either remove OPENSHIFT_VERSION, or make it local to the places it's really needed (only the `oc` download IIRC, everywhere else we should use the release-derived version since #1146)

Reply:
Thanks @hardys 

I'll check the other places as well, if it does not need dramatic changes, I can add it to this PR.

Reply:
I removed `OPENSHIFT_VERSION` usage only https://github.com/openshift-metal3/dev-scripts/blob/82293c97caddfd13829b97d13c110e389acf5fa4/common.sh#L93 remained.